From 08adb17f76c5b31053698f13c57caa27417185bb Mon Sep 17 00:00:00 2001
From: Shawn Landden <shawnlandden@gmail.com>
Date: Fri, 8 Jun 2012 00:28:44 -0700
Subject: [PATCH] disable march=native

---
 configure |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index a07063d..6e624fa 100755
--- a/configure
+++ b/configure
@@ -3991,8 +3991,8 @@ then
   printf 'checking for CPU optimization availability... '
   if printf 'main() {}' | $CC -xc -march=native -o config.tmp - >config.tmp 2>&1
   then
-    MYCFLAGS="-march=native $MYCFLAGS"
-    MYCXXFLAGS="-march=native $MYCXXFLAGS"
+#    MYCFLAGS="-march=native $MYCFLAGS"
+#    MYCXXFLAGS="-march=native $MYCXXFLAGS"
     printf 'yes\n'
   else
     printf 'no\n'
-- 
1.7.9.5

